support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/helpers/paymentSelection.jsx

```diff
@@ -32,8 +32,7 @@ const BILLING_PERIOD = {
     title: 'Monthly',
     singlePeriod: 'month',
     salesCopy: (displayPrice: string, saving?: string) => (
-      <span>14 day free trial, then
-        &nbsp;<strong>{displayPrice}</strong> a month for 12 months {saving && null}
+      <span>14 day free trial, then <strong>{displayPrice}</strong> a month for 12 months {saving && null}
```

Comment:
What is this `&& null` doing?

Reply:
this was a quick fix because the once I change the function to have a different number of variable, I need to create the corresponding type signature which I wasn't sure how to do